_get_started/installation/mac.md

```diff
@@ -1,16 +1,14 @@
 # Installing on macOS
 {:.no_toc}
 
-PyTorch can be installed and used on macOS. Depending on your system and compute requirements, your experience with PyTorch on a Mac may vary in terms of processing time. It is recommended, but not required, that your Mac have an NVIDIA GPU in order to harness the full power of PyTorch's [CUDA](https://developer.nvidia.com/cuda-zone) [support](https://pytorch.org/tutorials/beginner/blitz/tensor_tutorial.html?highlight=cuda#cuda-tensors).
-
-> Currently, CUDA support on macOS is only available by [building PyTorch from source](#mac-from-source)
+PyTorch can be installed and used on macOS. Depending on your system and GPU capabilities, your experience with PyTorch on a Mac may vary in terms of processing time. 
```

Comment:
This can be made more specific ? I think we are trying to mention about x86 system vs M1 system and support for MPS?
Or we want to add the MPS bits in a followup PR and we can clean it up later.

Reply:
If you have some wording in mind, please do not hesitate to suggest